### DISCUSSION TITLE
Ambiente colaborativo

### DISCUSSION THREAD

**Discussion by Juuhz** [frontendbr/forum#1461] [Discussão]
Fala Fronts, beleza?

Trabalho em uma empresa onde estamos iniciando um ambiente colaborativo, grupos de slack, eventos com tudo pago para algumas equipes, reuniões semanais, etc... 

Como estamos no começo dessa "transformação", alguns membros ainda tem alguma resistência e gostaria de fazer um networking com vocês para saber:

**Oque da certo na empresa de vocês?** 

Talvez possa trazer algum insight de vocês para a minha empresa, oque seria bem bacana.

**Comment by felipefialho** [comment]
@Juuhz Primeiro, você consegue explicar melhor o que seria essa resistência de alguns membros?
